web-local/config-local.yaml

```diff
@@ -56,6 +56,10 @@ dataCenter:
 systemOfRecord:
   migrationPhase: PRE_MIGRATION
   stashRoot: s3://emodb-us-east-1/stash/ci
+
+  # Optional property - to exclude the tables satisfying the condition from the daily Stash run.
+  stashBlackListTableConditionString: if(intrinsic(\"~table\":or(\"brandanswers:badgeoverride\",\"brandanswers:badgeoverrideimage\")))
```

Comment:
This example doesn't look like a condition string.  The surrounding "if()" is used only when defining role permissions.